src/Behat/MinkExtension/Context/MinkAwareContext.php

```diff
@@ -33,4 +34,11 @@ public function setMink(Mink $mink);
      * @param array $parameters
      */
     public function setMinkParameters(array $parameters);
+
+    /**
+     * Sets event dispatcher.
+     *
+     * @param HookDispatcher $dispatcher The dispatcher.
+     */
+    public function setDispatcher(HookDispatcher $dispatcher);
```

Comment:
this is impossible. Adding a new method in the interface is a BC break (and I don't want to go to MinkExtension 3.0 for now)